db/migrate/20230316050555_create_addresses.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAddresses < ActiveRecord::Migration[7.0]
+  def change
+    create_table :addresses do |t|
+      t.text :address
+      t.text :city
+      t.text :zip_code
+      t.text :state
+      t.text :country
```

Comment:
Is this missing a reference to the yardsale id? I may be mistaken, but based on the models it seems we want each address to be linked to a yardsale

Reply:
reference added